assets/react/v3/shared/atoms/Alert.tsx

```diff
@@ -0,0 +1,53 @@
+import { css } from '@emotion/react';
+
+import { borderRadius, colorTokens, spacing } from '@Config/styles';
+import type { IconCollection } from '@Utils/types';
+import { typography } from '@Config/typography';
+
+import SVGIcon from './SVGIcon';
+
+type AlertType = 'success' | 'warning' | 'danger' | 'info' | 'primary';
+
+interface AlertProps {
+	children: React.ReactNode;
+	type?: AlertType;
+	icon?: IconCollection;
+}
+
+const Alert = ({ children, type = 'warning', icon }: AlertProps) => {
+	return (
+		<div css={styles.wrapper({ type })}>
+			<SVGIcon style={styles.icon({ type })} name={icon as IconCollection} height={24} width={24} />
```

Comment:
Why do I need to cast the type explicitly here?

Reply:
As icon is optional it can be `undefined` so we need to cast the type explicitly. I forgot to do the conditional check. I will fix it on the next commit.